src/StatusQ/Controls/StatusSwitchTabButton.qml

```diff
@@ -8,6 +8,8 @@ import StatusQ.Core.Theme 0.1
 TabButton {
     id: statusSwitchTabButton
 
+    property int pixelSize: 15
```

Comment:
`fontPixelSize`? otherwise one might get the impression it's the size (height?) of the button itself